earlgrey/message_queue_connection.py

```diff
@@ -32,14 +32,19 @@ def __init__(self, amqp_target, route_key, username=None, password=None):
         self._async_info: MessageQueueInfoAsync = None
 
     async def connect(self, connection_attempts=None, retry_delay=None):
+        kwargs = {}
+        if connection_attempts is not None:
+            kwargs['connection_attempts'] = connection_attempts
+        if retry_delay is not None:
+            kwargs['retry_delay'] = retry_delay
+
         self._connection: RobustConnection = await aio_pika.connect_robust(
-            f"amqp://{self._amqp_target}",
+            host=self._amqp_target,
             login=self._username,
             password=self._password,
-            connection_attempts=connection_attempts,
-            retry_delay=retry_delay)
+            **kwargs)
+
         self._connection.add_close_callback(self._callback_connection_close)
-        self._connection.add_connection_lost_callback(self._callback_connection_lost_callback)
```

Comment:
Is this removed on `aio-pika`?

Reply:
It was remove on this PR : https://github.com/mosquito/aio-pika/pull/193

@eunsoo-icon @goldworm-icon 
We need to update where use '_callback_connection_lost_callback' in our project.